src/source/Ice/IceAgent.c

```diff
@@ -1774,6 +1774,57 @@ STATUS iceAgentCheckConnectionStateSetup(PIceAgent pIceAgent)
     return retStatus;
 }
 
+STATUS updateCandidateStats(PIceAgent pIceAgent, BOOL isRemote)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    CHK(pIceAgent != NULL, STATUS_NULL_ARG);
```

Comment:
Recommend also checking pIceAgent->pDataSendingIceCandidatePair not being null

CHK(pIceAgent != NULL && pIceAgent->pDataSendingIceCandidatePair != NULL, STATUS_NULL_ARG);

Reply:
Done